src/Catalyst.Core.Modules.Web3/Controllers/Handlers/EthWeb3RequestHandler.cs

```diff
@@ -78,10 +80,23 @@ public abstract class EthWeb3RequestHandler<TParam1, TParam2, TResult> : EthWeb3
         public override object Handle(string[] parameters, IWeb3EthApi api, IJsonSerializer serializer)
         {
             TParam1 param1 = Deserialize<TParam1>(parameters[0], serializer);
-            TParam2 param2 = Deserialize<TParam2>(parameters[0], serializer);
+            TParam2 param2 = Deserialize<TParam2>(parameters[1], serializer);
             return Handle(param1, param2, api);
         }
 
         protected abstract TResult Handle(TParam1 param1, TParam2 param2, IWeb3EthApi api);
     }
+
+    public abstract class EthWeb3RequestHandler<TParam1, TParam2, TParam3, TResult> : EthWeb3RequestHandlerBase
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Reduce the number of generic parameters in the 'EthWeb3RequestHandler' class to no more than the 2 authorized.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4585091)